src/FeatureDescriptors.jl

```diff
@@ -54,6 +54,12 @@ function label(D::Type{<:Descriptor})
     return Symbol(s)
 end
 
+# Allow sorting descriptors lexographically
+# NOTE: isless(hash(A), hash(B)) would be faster, but:
+# 1. we shouldn't need to sort that many descriptors
+# 2. this is more human readable/understandable
```

Comment:
When would it need to be understood? Does `hash` order lexographically as well? (If not, this makes more sense)

Reply:
I guess not

```julia
julia> using FeatureDescriptors

julia> abstract type DescriptorA <: Descriptor end

julia> abstract type DescriptorB <: Descriptor end

julia> isless(hash(DescriptorA), hash(DescriptorB))
false
```